### DISCUSSION TITLE
ctest is not found

### DISCUSSION THREAD

**Discussion by florenso** [microsoft/vscode-cmake-tools#3995] [Q&A] [answered]
Building works fine on my system but for some reason when i run the tests i it does not find the ctest.exe.

> [main] Building folder: C:/ontwikkeling/igtwebd/build/debug/windows 
[build] Starting build
[proc] Executing command: "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" --build C:/ontwikkeling/igtwebd/build/debug/windows --parallel 4 --target igtwebdtest --
[build] ninja: no work to do.
[driver] Build completed: 00:00:00.075
[build] Build finished with exit code 0
[proc] Executing command: "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\" -C Debug -T test --output-on-failure -R ^c\:\/ontwikkeling\/igtwebd$
[proc] The command: "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\" -C Debug -T test --output-on-failure -R ^c\:\/ontwikkeling\/igtwebd$ failed with error: Error: spawn C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\ ENOENT
[ctest] CTest finished with return code -4058
[ctest] Test results not found.

if i run `"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\ctest" --test-dir ./build/debug/windows/igtwebd/` it works just fine. 
strange that i need to add the --test-dir and the ctest in the path. any idea what is going on?

**Comment by gcampbell-msft** [comment, marked as answer]
@florenso Is there any specific way that you're defining the ctest executable? It looks like it's trying to invoke the bin folder rather than the ctest.exe itself.
